Detectors/HMPID/workflow/src/PedestalsCalculationSpec.cxx

```diff
@@ -229,11 +241,8 @@ void PedestalsCalculationTask::recordPedInDcsCcdb()
     o2::dcs::addConfigItem(pedestalsConfig, "Equipment" + std::to_string(e), (const char*)outBuffer);
   }
 
-  struct timeval tp;
-  gettimeofday(&tp, nullptr);
-  uint64_t ms = tp.tv_sec * 1000 + tp.tv_usec / 1000;
-  uint64_t minTimeStamp = ms;
-  uint64_t maxTimeStamp = ms + 1;
+  long minTimeStamp = o2::ccdb::getCurrentTimestamp();
+  long maxTimeStamp = o2::ccdb::getFutureTimestamp(60 * 60 * mDcsCcdbAliveHours * 1);
```

Comment:
This should be an absolute end-validity time in ms, i.e. `maxTimeStamp = minTimeStamp + 3600L * DcsCcdbAliveHours * 1000;`